internal/blueprint/customizations.go

```diff
@@ -64,6 +65,12 @@ type ServicesCustomization struct {
 	Disabled []string `json:"disabled,omitempty" toml:"disabled,omitempty"`
 }
 
+type FilesystemCustomization struct {
+	Name       string `json:"name,omitempty" toml:"name,omitempty"`
+	Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
+	MinSize    int    `json:"minsize,omitempty" toml:"size,omitempty"`
+}
```

Comment:
I don't know how elaborate we want to get (hopefully not very...) but for reference this is what pykickstart supports:
https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#id46

Reply:
Thanks Brian! I think we'll aim for as minimal as possible for now and extend it when we must, but good to keep what kickstart does in mind.